cherryontech/ywgc
# Add picture on home page

## Iteration - 1

https://ywgc.atlassian.net/browse/YWGC-24?atlOrigin=eyJpIjoiZDA0OTMzYzgwZGJkNDk3N2FkNzI0NDVmMDBiODlmNzQiLCJwIjoiaiJ9

## Change background of the website, using a picture from figma on home page

## Manual testing instructions
Accessibility - Not sure how to add alt text for the background image at the moment will fix it once merge.
Responsive - Check if the background image is responsive to different screen sizes.


## Gotchas/What I learned
Accessibility is really important for the website, same goes for alt-text. Having a good alt-text could offer user a much better experience.
